adapters/adorsys-integ-adapter/pom.xml

```diff
@@ -22,14 +26,48 @@
             <version>${project.version}</version>
         </dependency>
 
+
         <!-- test dependencies -->
 
+        <dependency>
```

Comment:
There should be already a transitive dependency from service-api-adapter.

Reply:
but they are not, rechecked

Reply:
What do you mean by "not rechecked"?

Reply:
```
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ adorsys-integ-adapter ---
[INFO] de.adorsys.xs2a.adapter:adorsys-integ-adapter:jar:0.0.12-SNAPSHOT
[INFO] +- de.adorsys.xs2a.adapter:xs2a-adapter-service-api-adapter:jar:0.0.12-SNAPSHOT:compile
[INFO] |  +- de.adorsys.xs2a.adapter:xs2a-adapter-service-api:jar:0.0.12-SNAPSHOT:compile
[INFO] |  |  +- de.adorsys.xs2a.adapter:xs2a-adapter-generated-api:jar:0.0.12-SNAPSHOT:compile
[INFO] |  |  |  \- javax.annotation:javax.annotation-api:jar:1.3.2:compile
[INFO] |  |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.11.0:compile
[INFO] |  |  \- org.slf4j:slf4j-api:jar:1.7.30:compile
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.12:compile
[INFO] |  |  +- org.apache.httpcomponents:httpcore:jar:4.4.13:compile
[INFO] |  |  \- commons-codec:commons-codec:jar:1.14:compile
[INFO] |  +- org.apache.httpcomponents:httpmime:jar:4.5.12:compile
```

Reply:
i've commented those dependencies and I've run `mvn clean install`
```
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   AdorsysAccountInformationServiceWireMockTest.beforeAll:58 » NoClassDefFound Co...
[ERROR]   PaymentInitiationServiceWireMockTest.beforeAll:62 » NoClassDefFound org/apache...
[INFO] 
[ERROR] Tests run: 30, Failures: 0, Errors: 2, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 6.157 s (Wall Clock)
[INFO] Finished at: 2020-07-10T08:08:56+03:00
[INFO] ------------------------------------------------------------------------

```

Reply:
`httpclient` dependency was removed